project_proposals/RCM OSS.md

```diff
@@ -0,0 +1,149 @@
+# Questionnaire for RCM OSS intending to join the OpenRail Association
+
+## What is the project's name?
+
+RCM OSS - Rail Condition Monitoring Open Source Software
+
+## Describe the project. What does the project do, why is it valuable, where does it come from?
+
+RCM by SBB comprises a suite of products for rail condition monitoring. Based on the open source data format RCM-DX (RCM Data eXchange), RCM enables easy accessibility and visualisation of railway diagnostic data.
+
+The aim of RCM OSS is to make rail condition data easy to access, store and visualize. Therefore, RCM OSS is beneficial to railway companies as well as measurement system suppliers.
+
+## Who are the maintainers of the project (these will be the primary contacts for the OpenRail Association)?
+
+* Stephanie Schalbetter: stephanie.schalbetter@sbb.ch
+* Jean-Frédéric Bonjour: jean-frederic.bonjour@sbb.ch
+
+## Which organizations are sponsoring/contributing to the project?
+
+Swiss Federal Railways SBB
+
+## Where is the code hosted?
+
+* [Viewer (Freeware)](https://github.com/SchweizerischeBundesbahnen/rcm-dx/releases)
+* [RCM DX Format Specification](https://github.com/SchweizerischeBundesbahnen/rcm-dx)
+* [Example Data Track (100 MB)](https://sbb-wcms-big-files-server-prod.s3.eu-central-1.amazonaws.com/RCM-DX_Files/20220428-114946_DFZ00_TGMS_v003_created-20220429-065432.rcmdx)
+* [Example Data Position (100 MB)](https://sbb-wcms-big-files-server-prod.s3.eu-central-1.amazonaws.com/RCM-DX_Files/20220428-114946_DFZ00_TLS_v001_created-20220429-065432.rcmdx)
+* [Example Data Track Inspection (167 MB)](https://sbb-wcms-big-files-server-prod.s3.eu-central-1.amazonaws.com/RCM-DX_Files/20220428-114946_DFZ00_TSIMS-RAW-COMP_v001_created-20220429-065432.rcmdx)
+* [Example Data Video (937 MB)](https://sbb-wcms-big-files-server-prod.s3.eu-central-1.amazonaws.com/RCM-DX_Files/20220428-114946_DFZ00_TVS_v001_created-20220429-065432.rcmdx)
+
+## What is the project's main license?
+
+Eclipse Public License 2.0
+
+## What other licenses does the project use, e.g. for included 3rd party code?
+
+Code is not Open Source:
+
+* BSD 3
+* BSD
+* GPL v3
+* Apache License 1.1 and 2.0
+* LGPLv3 with  static linking  exception
+* MPL v2
+* ISC
+* MIT
+* LGPL V2.1+
+* NUnit License
+* The BSD 2 -  Clause License
+* New BSD License (BSD)
+* LGPL
+* http:// x264licensing.co m/ Professional License
```

Comment:
are the white spaces in the link intentional?
or is this a place holder for https://x264.org/en/ ?